### ISSUE TITLE
Games issues

### ISSUE THREAD

**Issue by luigiplus** [ata4/angrylion-rdp-plus#20]
This is my current emulation machine: I5 3570 , 8GB RAM, R9 270X  2GB,  windows 10 creators update.
All drivers updated.
Project64 2.4.0.9999 default plugins and angrylion-plus r4:

banjo kazooie and tooie graphic tremble. ( angrylion-plus r3 work good)

donkey kong 64 crash in logo rare. ( angrylion-plus r3 work good)

F1 WORLD GRAND PRIX in start race show error SP_DRAM_ADDR_REG not in RDram space

V-Rally Edition 99 (U) no boot project64  stopped working

**Comment by ata4** [comment]
Hm... DK64 is indeed crashing, but only when using 2.4.0.9999. With the same build, I have strange frame duplication issues with BK and BJ, but both for r3 and r4. With 2.4.0.452, I don't have any problems with these games. Not sure which side is to blame, I suspect there's a buffer overflow somewhere and it doesn't affect 2.4.0.452 severely. I'll try a `git bisect` to find the commit that may have caused this issue.

**Comment by loganmc10** [comment]
@ata4 it's probably Project64's fault, see:

https://github.com/project64/project64/issues/1022#issuecomment-326142857

**Comment by ata4** [comment]
Yes, probably. I have the same issues with vanilla Angrylion.

**Comment by SigmaVirus** [comment]
@ata4 
Yes I agree, Project64 2.4.0.9999 is a real mess and sometimes it just crashes for looking at it wrong.

Where do you download 2.4.0.452? I can't see or find it anywhere, is it a patron build only for supporters?
